unic/ucd/name/tests/basic_tests.rs

```diff
@@ -13,12 +13,37 @@ extern crate unic_ucd_name;
 use unic_ucd_name::Name;
 
 #[test]
-fn basic_tests() {
+fn test_name_str() {
     assert_eq!(
         Name::of('A').expect("No name for A").to_string(),
         "LATIN CAPITAL LETTER A"
     );
+    assert_eq!(
+        Name::of('곲').expect("No name for 곲").to_string(),
+        "HANGUL SYLLABLE ACF2" // FIXME: use decomposed jamo short names instead
+    );
+    assert_eq!(
+        Name::of('言').expect("No name for 言").to_string(),
+        "CJK UNIFIED IDEOGRAPH-8A00"
+    );
+    assert_eq!(
+        Name::of('\u{17005}').expect("No name for U+17005").to_string(),
+        "TANGUT IDEOGRAPH-17005"
+    );
+    assert_eq!(
+        Name::of('\u{1B175}').expect("No name for U+1B175").to_string(),
+        "NUSHU CHARACTER-1B175"
+    );
+    assert_eq!(
+        Name::of('理').expect("No name for 理").to_string(),
+        "CJK COMPATIBILITY IDEOGRAPH-F9E4"
+    );
     assert_eq!(Name::of('\u{10FFFF}'), None);
+}
 
+#[test]
+fn test_name_cmp() {
     assert!(Name::of('A') < Name::of('B'));
+    assert!(Name::of('言') < Name::of('A'));
```

Comment:
Would you please expand the test here to compare `Name::of()` AND their `to_string()` for any pair of sample?

Reply:
Expanded the test with more comparison between samples and their `to_string()` comparison.